README.md

```diff
@@ -462,6 +462,17 @@ following
 4. Now paste the API key to your `.env` file into the `DAILY_API_KEY` field in your `.env` file.
 5. If you have the [Daily Scale Plan](https://daily.co/pricing) set the `DAILY_SCALE_PLAN` variable to `true` in order to use features like video recording.
 
+### Obtaining Basecamp Client ID and Secret
+
+1. Visit the [37 Signals Integrations Dashboard](launchpad.37signals.com/integrations) and sign in.
+2. Register a new application by clicking the Register one now link.
+3. Fill in your company details.
+4. Select Basecamp 4 as the product to integrate with.
+5. Set the Redirect URL for OAuth `<Cal.com URL>/api/integrations/basecamp3/callback` replacing Cal.com URL with the URI at which your application runs.
+6. Click on done and copy the Client ID and secret into the `BASECAMP3_CLIENT_ID` and `BASECAMP3_CLIENT_SECRET` fields.
+7. Go to the path `packages/app-store/basecamp3/lib/constants.ts` and edit the userAgent variable to `{your_domain} {support_email}` (VERY IMPORTANT).
```

Comment:
Sorry @Jonathan1599, I meant to create a new environment variable along with `BASECAMP3_CLIENT_ID` and `BASECAMP3_CLIENT_SECRET` to have this setup directly, like `BASECAMP3_USER_AGENT`. Don't forget to add it to the `zod.ts` file as well so it can be changed from the apps section in admin settings. Thanks @joeauyeung for the pointer.

Reply:
Yeah, this is a better solutions, thanks! Will implement